lib/tosdp.js

```diff
@@ -7,8 +7,10 @@ exports.toSessionSDP = function (session, opts) {
     var sid = opts.sid || session.sid || Date.now();
     var time = opts.time || Date.now();
 
+    var type = (session.type && 'type: ' + session.type + ', ') || '';
+
     var sdp = [
-        'v=0',
+        type + 'sdp: v=0',
```

Comment:
this looks odd?

Reply:
Sorry, I meant to call this out but forgot. In chrome we get something like `type: offer, sdp: v=0` or `type: answer, sdp: v=0` but ff just gets `v=0` like what was there originally. I can revert this if needed, I wasn't sure the significance here.

Reply:
that is the rtcsessiondescription object which has type and sdp properties. But even that is a proper js object and this isn't?

Reply:
Sorry, you're right. I'm going crazy. I'll revert this bit.